### PR TITLE
Adding BigQuery label length req

### PR DESCRIPTION
**Pull request by leahwicz**
## Description & motivation
Adding in a note about BigQuery enforcing a max length on labels. This has been a new error shown in dbt that people have hit. It would be ideal if they knew the limitation beforehand though so they can avoid the issue all together. 

Example issue: https://github.com/dbt-labs/dbt-bigquery/issues/202
Code where error message was added: https://github.com/dbt-labs/dbt-core/pull/3703

### CODE REVIEW

**Review by McKnight-42 — APPROVED** [review]
Looks like a good notifier

File: website/docs/reference/resource-configs/bigquery-configs.md [hunk 1]
```diff
@@ -294,6 +294,8 @@ models:
 dbt supports the specification of BigQuery labels for the tables and <Term id="view">views</Term> that it creates. These labels can be specified using the `labels` model config.
 
 The `labels` config can be provided in a model config, or in the `dbt_project.yml` file, as shown below.
+  
+**Note:** Per BigQuery requirements, both key-value pair entries for labels have a maximum length of 63 characters.
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/bigquery-configs.md]: Looks good, just adding our note format:

```suggestion
:::note
BigQuery requires that both key-value pair entries for labels have a maximum length of 63 characters.
:::
```

**Review by runleonarun — CHANGES_REQUESTED** [review]

File: website/docs/reference/resource-configs/bigquery-configs.md [hunk 2]
```diff
@@ -294,6 +294,10 @@ models:
 dbt supports the specification of BigQuery labels for the tables and <Term id="view">views</Term> that it creates. These labels can be specified using the `labels` model config.
 
 The `labels` config can be provided in a model config, or in the `dbt_project.yml` file, as shown below.
+  
+:::note
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/bigquery-configs.md]:
```suggestion
:::info Note
```

**Review by runleonarun — APPROVED** [review]
🚢 it!

**Pull request merged**